lang/cs/Org.Apache.REEF.Wake/Time/Runtime/RuntimeClock.cs

```diff
@@ -73,6 +74,11 @@ public sealed class RuntimeClock : IClock
             _runtimeStartHandler = runtimeStartHandler;
             _runtimeStopHandler = runtimeStopHandler;
             _idleHandler = idleHandler;
+            ++NumberOfInstantiations;
```

Comment:
@markusweimer suggested to keep these even after debug, to alert us of this issue when it happens. But now the underlying cause seems to be fixed in REEF-217. @markusweimer, do you think we still want to keep this?